sandbox/tailwind-app/src/index.css

```diff
@@ -15,7 +15,9 @@
   width: 100%;
 }
 :focus-visible {
-  outline: 2px auto var(--hpe-focus-ring-color);
+  outline: var(--hpe-focus-indicator-outline);
```

Comment:
Are similar changes needed in the native and/or Grommet apps?

Reply:
I'll update in native-app here. Will need to assess how to drive this from Grommet theme/if any enhancements are necessary.